hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/util/RowDataProjection.java

```diff
@@ -87,4 +88,8 @@ public Object[] projectAsValues(RowData rowData) {
     }
     return values;
   }
+
+  protected @Nullable Object rewriteVal(int pos, @Nullable Object val) {
+    return val;
```

Comment:
`rewriteVal` => `getVal `, usually we do not overwrite impl methods, but only abstract methods, the override is not very friendly for base class performance.

Reply:
renamed